stdlib/in_channel.mli

```diff
@@ -165,3 +180,19 @@ val isatty : t -> bool
     [false] otherwise.
 
     @since 5.1 *)
+
+(** {1:examples Examples}
+   Reading the contents of a file:
+    {[
+    # let text = In_channel.with_open_text "./example.txt"
+                         (fun ic -> In_channel.input_all ic)
+    val text : string = "..."
+    ]}
```

Comment:
This is not a good example. To read the contents of a file you should not use `text` mode. Here is my suggestion:
```
Reading the contents of a file:
{[
let read_file file = In_channel.with_open_bin file In_channel.input_all
]}
```

Reply:
Updated examples to use bin instead